src/UI/Tables/Stateful.elm

```diff
@@ -1421,12 +1429,40 @@ desktopView renderConfig prop opt =
                 columns
                 selectionHeader
     in
-    Keyed.column
-        [ Element.spacing 2
-        , Element.width opt.width
-        , Element.paddingEach padding
-        ]
-        (headers :: rows ++ footer)
+    case state.paginator of
+        Just paginator ->
+            Keyed.column
+                [ Element.width opt.width
+                , Element.height opt.height
+                , Element.paddingEach padding
+                ]
+                [ Keyed.column
+                    [ Element.spacing 2
+                    , Element.height Element.fill
+                    , Element.width Element.fill
+                    , Element.alignTop
+                    , Element.scrollbars
+                    ]
+                    (headers :: rows)
```

Comment:
should the headers scroll?

Reply:
as we chatted about in slack, fixing the header on top and making it scroll horizontally with the content has some usability issues if we do it the usual way (the vertical scrollbar ends on the right side of the content, forcing the user to scroll horizontally to reach it)

there are hacky ways to sync the header's position with the horizontal scrollbar, but doing it without js will cause performance issues, so it's better if I postpone it (and paack-ui has no js, so I'll be "breaking" a good package for that :/)